arangod/Scheduler/SupervisedScheduler.cpp

```diff
@@ -155,11 +156,13 @@ SupervisedScheduler::SupervisedScheduler(application_features::ApplicationServer
       _jobsSubmitted(0),
       _jobsDequeued(0),
       _jobsDone(0),
-      _minNumWorkers(minThreads),
-      _maxNumWorkers(maxThreads),
-      _maxFifoSize(maxQueueSize),
-      _fifo1Size(fifo1Size),
-      _fifo2Size(fifo2Size),
+      _minNumWorker(minThreads),
+      _maxNumWorker(maxThreads),
+      _maxFifoSizes{maxQueueSize, fifo1Size, fifo1Size, fifo3Size},
+      _ongoingLowPriorityLimit(static_cast<std::size_t>(ongoingMultiplier * _maxNumWorker)),
+      _wakeupQueueLength(5),
+      _wakeupTime_ns(1000),
+      _definitiveWakeupTime_ns(100000),
```

Comment:
Wondering why these variables are reintroduced in this PR.
I deleted them previously, because they did nothing. :thinking:

Reply:
Merge artifact I think. Removed again.